data/phones/nob_phonemic.phones

```diff
@@ -0,0 +1,67 @@
+# Based on
+#
+### CONSONANTS ###
+p
+b
+m
+f
+v  # Allophone of /ʋ/.
+ʋ
+t
+d
+n
+r
+ɾ  # Less common (but apparently, more phonetically accurate) transcription of /r/.
+s
+l
+ʃ  # Less common transcription of /ʂ/.
+ʈ  # Underlyingly /rt/.
+ɳ  # Underlyingly /rn/.
+ʂ
+ɽ
+ç
+j
+k
+ɡ
+ŋ
+h
+# Syllabic consonants
+n̩
+### VOWELS ###
+i  # Less common transcription of /ɪ/.
```

Comment:
Huh so it looks like they couldn't decide whether to transcribe quality differences for quantity differences...this problem comes up in the transcription of pretty much every Germanic language I know of. (In English it's pretty clear monophthong length is tense vs. lax nowadays; in German it's not obvious to me that closed-syllable monophthongs are really laxer but they are usually transcribed like that, and so on...)

Reply:
Yep, that's exactly it. Although there wasn't a clear preference for one or the other overall, it seemed – for some pairs, they preferred length distinctions (_e.g._, /e/ was more common than /ɛ/), while for other they preferred tenseness (_e.g._, /ɪ/ was more common than /i/). But in a lot of cases, both styles were very common. For example, <ɪ> occurs 157 times, but <i> occurs 112 times – about 10% of all the entries!

Reply:
I think this is something we could fix by bot.

Possibly a dumb question, but did you see any correlation with whether the syllable was closed or open?

Reply:
I looked through the TSV file a bit to compare /ɪ/ and /i/, and I didn't really notice any patterns. Both seem to occur in the same contexts, as far as I can tell